index.bs

```diff
@@ -679,6 +667,20 @@ The [=reading change treshold|treshold=] value depends on the surrounding softwa
 environment constraints, e.g., software power consumption optimizations or the underlying
 [=device sensor=]'s accuracy.
 
+## Conditions to expose sensor readings ## {#concepts-can-expose-sensor-readings}
+
+In order to check if user agent <dfn>can expose sensor readings</dfn> following criteria may be used:
```

Comment:
maybe tell that user agent is to define the necessary criteria  to meet the [=conditions=] , i.e.  which [=mitigation strategies=] must be applied and it depends on the underlying device' form factor or so.. The example is below